documentation/src/main/asciidoc/topics/proc_displaying-metrics-st-operation.adoc

```diff
@@ -0,0 +1,33 @@
+[id='displaying-metrics-st-operationadoc_{context}']
+= Displaying metrics during a state transfer operation
+
+You can export time metrics for clustered caches that {brandname} redistributes across nodes. 
+
+A state transfer operation occurs when a clustered cache topology changes, such as a node joining or leaving a cluster.
+During a state transfer operation, {brandname} exports metrics from each cache, so that you can determine a cache's status.
+A state transfer exposes attributes as properties, so that {brandname} can export metrics from each cache.
+
+[NOTE]
+====
+You cannot perform a state transfer operation in invalidation mode. 
+====
+
+{brandname} generates time metrics that are compatible with the REST API and the JMX API. 
+
+.Prerequisites
```

Comment:
Consistency... Please check the prerequisites in the operator guide and try to keep it in a similar tone. 
For example, Have {brandname} metrics configured.

Reply:
Let's move the links to the additional resources?

Reply:
We can reference the assembly that includes both modules. WDYT?https://infinispan.org/docs/stable/titles/configuring/configuring.html#statistics-jmx

Reply:
Applied all these changes, but I'll keep the two links for better user awareness.